src/GridFS/Bucket.php

```diff
@@ -695,7 +695,7 @@ private function getFilesNamespace()
      * @return stdClass
      * @throws InvalidArgumentException
      */
-    private function getRawFileDocumentForStream($stream)
+    private function getRawFileDocumentForStream($stream): stdClass
```

Comment:
I realize this is a `private` method so it's not really a concern here, but if we ever think about adding `stdClass` as a return type hint for a public method I think we should consider using `object` just to keep our options open. I'm thinking about `ClientEncryption::rewrapManyDataKey()` as I write this :)

Reply:
Absolutely - I'd have to take another look (and will do so as part of the psalm integration), but this is one case where we may want to change it to `object` anyways.

Reply:
No objections if you'd like to just use `object` here.

Reply:
Changed this to `object` to avoid relying on `stdClass`.